README.md

```diff
@@ -24,6 +24,12 @@ The .exe file is an installer - just open it and install the native application.
 
 ### Installation on Linux
 
+Run script:
+
+`wget -O - https://raw.githubusercontent.com/Virusmater/firefox-webserial/master/install.sh | bash`
```

Comment:
Is is possible to use `curl` for this? It is more likely to have curl installed by default on a Linux distro than wget. Also curl is used anyway in the .sh script.

The URL has to be adjusted to point to the original repo :smile:

Reply:
Done